index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 global.D2LAccessTokenExpiresAt = 0;
+global.D2LOAuth2Disabled = false;
```

Comment:
For testing, yeah you could add methods to `module.exports` called something like `_setAccessTokenExpiry()` and `_setOAuthIsDisabled()` that just manipulate those local variables.